### PR TITLE
Rounding Fix

### PR DESCRIPTION
**Pull request by aparthibanpaypal**

### CODE REVIEW

**Review by shanavas123 — DISMISSED** [review]

**Review by SaiNarendran — APPROVED** [review]

File: Common/Sources/Extensions/String.swift [hunk 1]
```diff
@@ -84,17 +84,12 @@ public extension String {
...
-        }
-        let numbers = self.split(separator: ".")
-        if numbers.count == 2 {
-            let decimals = numbers[1]
-            if decimals.count > 4 {
-                return String(self.dropLast(decimals.count - 4))
-            }
-        }
-        return self
+        let number = self.formatAmountToDouble()
```
Comment (jsaini-hw) [inline Common/Sources/Extensions/String.swift:87]: keep guard statement to check if self is not empty

Comment (aparthibanpaypal) [reply Common/Sources/Extensions/String.swift:87]: Updated

**Review by SaiNarendran — APPROVED** [review]

**Review by shanavas123 — DISMISSED** [review]

File: Common/Sources/Extensions/String.swift [hunk 2]
```diff
@@ -84,17 +84,14 @@ public extension String {
...
-        let numbers = self.split(separator: ".")
-        if numbers.count == 2 {
-            let decimals = numbers[1]
-            if decimals.count > 4 {
-                return String(self.dropLast(decimals.count - 4))
-            }
-        }
-        return self
+        guard !self.isEmpty
+        else { return "0" }
```
Comment (rbanikPP) [inline Common/Sources/Extensions/String.swift:88]: Is there any specific reason, this return statement is changed?

Comment (jsaini-hw) [reply Common/Sources/Extensions/String.swift:88]: It should still be "" and not 0

Comment (aparthibanpaypal) [reply Common/Sources/Extensions/String.swift:88]: Ok, will update

**Review by shanavas123 — DISMISSED** [review]

**Review by SaiNarendran — DISMISSED** [review]

**Review by rbanikPP — APPROVED** [review]

**Review by rbanikPP — DISMISSED** [review]

**Review by jsaini-hw — APPROVED** [review]

**Review by SaiNarendran — APPROVED** [review]

**Review by SaiNarendran — APPROVED** [review]

**Review by shanavas123 — APPROVED** [review]

**Review by rbanikPP — APPROVED** [review]

**Review by jsaini-hw — APPROVED** [review]

**Pull request merged**